src/main/java/frc/robot/commands/oldClimber/calibrateClimber.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.oldClimber;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.OldClimber;
+import static frc.robot.settings.Constants.Climber.*;
+
+
+public class calibrateClimber extends CommandBase {
+  OldClimber climber;
+
+  /** Creates a new calibrateClimber. */
+  public calibrateClimber(OldClimber climber) {
+    this.climber = climber;
+    
+    addRequirements(climber);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    climber.climberIn(CLARM_SPEED_CALIBRATE);
```

Comment:
```suggestion
    climber.climberIn(ARM_CALIBRATION_SPEED);
```
Rename this variable, `CLARM` is not specific. If this is a static import `ARM_CALIBRATION_SPEED` would be a fine name.